generators/improve_report.py

```
@@ -0,0 +1,378 @@
+"""
+            for i, prev in enumerate(request.prev_data, 1):
+                context += f"""
+### {i}차 특약:
+제목: {prev.title}
+내용: {prev.content}
+당시 대화: {prev.messages}
+"""
+            
+            context += f"""
+# 개선 목표: 사전조사 정보와 히스토리를 참고하여 현재 특약을 최근 대화 내용에 맞게 추가 개선해주세요.
+"""
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)